### DISCUSSION TITLE
Do World Eaters work on Purpur?

### DISCUSSION THREAD

**Discussion by dti131** [PurpurMC/Purpur#935] [__Dismissed__] [answered]
We all know that Paper and Purpur mess with redstone mechanics. My question is, does anyone know if a world eater like in [this video](https://www.youtube.com/watch?v=Pzo6ZgO3elM) works on purpur? And if it doesn't, what purpur settings would allow it to?
And there's no questions category so I'm just putting it here

**Comment by granny** [comment, marked as answer]
Purpur doesn't mess with redstone mechanics in any way, so we don't have any config options for you to enable/disable. I've never built that machine so I can't say, but if you run into any issues then it's because of something upstream. 

> And there's no questions category so I'm just putting it here

Feel free to join our discord for that next time https://purpurmc.org/discord
